public/javascripts/editneumescript.js

```diff
@@ -85,27 +90,18 @@ function initEditNeume(neume) {
         }
         //Append the button to the image card.
         imageCard.appendChild(buttonDeleteImage);
-        $(buttonDeleteImage).click(function() {
-            document.getElementById("imageToDelete" + neume._id).value = element;
-            $("#deleteImagebutton" + neume._id).click();
+        buttonDeleteImage.onclick = ('click', function() {
+            if (imagesToDelete == 'none') {
+                imagesToDelete = [];
+            }
+            imagesToDelete.push(element);
+            document.getElementById("imageToDelete" + neume._id).value = imagesToDelete;
             //Delete the image :
             canvas.style.display = "none";
             buttonDeleteImage.style.display = "none";
         })
```

Comment:
Does this work? Typically `onclick` is assigned to a function rather than what seems to be a tuple here. Also, is there a reason to set an attribute rather than use `addEventListener`?